nebula-logger/main/log-management/lwc/logEntryEventStream/logEntryEventStream.js

```diff
@@ -104,44 +97,38 @@ export default class LogEntryEventStream extends LightningElement {
 
     onToggleStream() {
         this.isStreamEnabled = !this.isStreamEnabled;
-        if (this.isStreamEnabled) {
-            this.createSubscription();
-        } else {
-            this.cancelSubscription();
-        }
+        // eslint-disable-next-line
+        this.isStreamEnabled ? this.createSubscription() : this.cancelSubscription();
```

Comment:
let me know what you think - when I wrote this, I thought it was nice, but then eslint hated it so ...

Reply:
Yeah, I like this approach - I also think it looks nice, so let's stick with this.